File: src/main/java/ordilov/lissn/member/application/port/in/UpdateMemberCommand.java
package ordilov.lissn.member.application.port.in;

import ordilov.lissn.member.domain.MemberInfo.UpdateMemberInfo;
import org.springframework.web.multipart.MultipartFile;

public interface UpdateMemberCommand {

  UpdateMemberInfo updateName(Long id, String name);

  UpdateMemberInfo updatePicture(Long id, MultipartFile imageFile);

  void deleteMember();
}

File: src/main/java/ordilov/lissn/member/application/port/in/UpdateMemberCommand.kt
package ordilov.lissn.member.application.port.`in`

import ordilov.lissn.member.domain.UpdateMemberInfo
import org.springframework.web.multipart.MultipartFile

interface UpdateMemberCommand {

    fun updateName(id: Long, name: String): UpdateMemberInfo

    fun updatePicture(id: Long, imageFile: MultipartFile): UpdateMemberInfo

    fun deleteMember()
}